server/athenian/api/internal/miners/jira/issue.py

```diff
@@ -888,6 +960,34 @@ def hint_epics(q):
     return df
 
 
+@sentry_span
+async def _query_jira_raw_by_keys(
+    keys: Sequence[str],
+    columns: list[InstrumentedAttribute],
+    jira_config: JIRAConfig,
+    mdb: Database,
+    cache: Optional[aiomcache.Client],
+) -> pd.DataFrame:
+    where = [
+        Issue.acc_id == jira_config.acc_id,
+        Issue.project_id.in_(jira_config.projects),
+        Issue.is_deleted.is_(False),
+        Issue.key.progressive_in(keys),
+    ]
+    issue_status_join = sa.join(
```

Comment:
It's better to do outerjoin here. Otherwise, the planner can do smth nasty.

Reply:
btw, is it to get the status category?

Reply:
It's needed by `_validate_and_clean_issues` which I was not calling, now I do